bff/solid/src/http.tsx

```diff
@@ -336,7 +337,9 @@ export async function queryFilterOptions(): Promise<boolean> {
     "" //
   }&filterdimselected=${encodeURIComponent(
     "", //
-  )}`;
+  )}&dbidentifier=${
+    state.instance_active.dbIdentifier //
```

Comment:
nit.

```suggestion
    state.instance_active.dbIdentifier
```

Reply:
That's there on purpose to keep the lines from being unwrapped back into 1 line. It makes the logic more clear this way by enabling the formatting to show the structure easily.